site/docs/skylark/performance.md

```diff
@@ -423,7 +423,7 @@ $ bazel $(STARTUP_FLAGS) dump --skylark_memory=$HOME/prof.gz
 Next, we use the `pprof` tool to investigate the heap. A good starting point is
 getting a flame graph by using `pprof -flame $HOME/prof.gz`.
 
-  You can get `pprof` from https://github.com/google/pprof.
+  You can get `pprof` from (https://github.com/google/pprof)[https://github.com/google/pprof].
```

Comment:
I think we want to switch `()[]` to `[]()`?

The first looks like (https://github.com/google/pprof)[https://github.com/google/pprof] while the second looks like [https://github.com/google/pprof](https://github.com/google/pprof).

Reply:
Yeah that's it 🙄